docs/fields/upload.mdx

```diff
@@ -27,6 +27,7 @@ keywords: upload, images media, fields, config, configuration, documentation, Co
 | ---------------- | ----------- |
 | **`name`** *         | To be used as the property name when stored and retrieved from the database.  |
 | **`*relationTo`** *  | Provide a single collection `slug` to allow this field to accept a relation to. <strong>Note: the related collection must be configured to support Uploads.</strong> |
+| **`maxDepth`**       | Sets a number limit on iterations of related documents to populate when queried. |
```

Comment:
We should link to the [Depth](https://payloadcms.com/docs/getting-started/concepts#depth) docs here.